src/client/client-legacy.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 The boardgame.io Authors
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import { error } from '../core/logger';
+import { SocketIO } from './transport/socketio';
+import { Local } from './transport/local';
+
+export function legacyCoerceMultiplayerOption(opts) {
```

Comment:
I'd say let's get rid of this (i.e. force all users to explicitly name the transport).